.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,12 +24,12 @@ Checklist
 
 <!--- Complete an item by checking it: [x] -->
 
-- [ ] All tests are passing
-- [ ] If necessary, new tests were created to address changes in PR, and tests are passing
+- [ ] All previous tests are passing
+- [ ] Tests were created to address all code changes in this PR (including bug fixes), and tests are passing
```

Comment:
let's keep this short. i suggest:

- [ ] All tests are passing
- [ ] All code changes meet testing guidelines (if applicable)

Reply:
I like the above, though I would write `(incl. bug fixes)`.

Reply:
"incl" is an uncommon abbreviation and is likely to be confusing for people whose first language is not English, so I'd avoid it. Also, not all changes to code will require _new_ tests to be written, thus the original phrasing, "If necessary." My suggestion is "Tests were updated or created to address changes in this PR."

Reply:
Okay, sounds reasonable.